share_extension/icons/excel.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {View} from 'react-native';
+import Svg, {
+    G,
+    Path
+} from 'react-native-svg';
+
+const ExcelSvg = ({height, width}) => { //eslint-disable-line
```

Comment:
These other icon components are still disabling the style guide. Also, shouldn't they be PureComponents?

Reply:
I don't see the need to add everything that a pureComponent includes, this can be a functional component without problem.

The style is disabled cause it ask the properties to be defined, I don't really think we need it, but if you think we should I can probably add those and even convert this to PureComponent

Reply:
Yeah, we should define them since we shouldn't be bypassing the style guide frequently.